pkg/pgmodel/pgx.go

```diff
@@ -198,11 +199,27 @@ func NewPgxIngestor(c *pgxpool.Pool) (*DBIngestor, error) {
 func newPgxInserter(conn pgxConn, cache MetricCache, cfg *Cfg) (*pgxInserter, error) {
 	cmc := make(chan struct{}, 1)
 
+	maxProcs := runtime.GOMAXPROCS(-1)
+	if maxProcs <= 0 {
+		maxProcs = runtime.NumCPU()
+	}
+	if maxProcs <= 0 {
+		maxProcs = 1
+	}
+
+	// we leavel one connection per-core for other usages
```

Comment:
what does this mean? It would have been 5 otherwise? The comment makes little sense

Reply:
@cevian yup, otherwise we'd have 5 workers, one per-connection. I'm concerned that if we do that new metrics would starve when trying to create tables. (alternatively, we could create `5*maxprocs - 1` workers, and only leave one for other tasks)

Reply:
I see that fine. But lets make 5 a variable somewhere that is used in both setting pool size and this. Right now there is nothing keeping the two values in sync.